{
    "summary": {
        "outcome": "Failed",
        "testsRan": 1,
        "passing": 0,
        "failing": 1,
        "skipped": 0,
        "passRate": "0%",
        "failRate": "100%",
        "testStartTime": "Dec 20, 2020 4:34 PM",
        "testExecutionTime": "114 ms",
        "testTotalTime": "114 ms",
        "commandTime": "5615 ms",
        "hostname": "https://mindful-fox-nzaknp-dev-ed.my.salesforce.com",
        "orgId": "00D4x00000312ujEAA",
        "username": "henrique_nitatori@mindful-fox-nzaknp.com",
        "testRunId": "7074x000005tebA",
        "userId": "0054x0000027d4KAAQ"
    },
    "tests": [
        {
            "attributes": {
                "type": "ApexTestResult",
                "url": "/services/data/v50.0/tooling/sobjects/ApexTestResult/07M4x0000008n4hEAA"
            },
            "Id": "07M4x0000008n4hEAA",
            "QueueItemId": "7094x000000YJGxAAO",
            "StackTrace": "Class.EventRepository.create: line 8, column 1\nClass.EventRepositoryTest.givenCreateWhenCreateEventThenExpectedListOfEvent: line 14, column 1",
            "Message": "System.DmlException: Insert failed. First exception on row 0 with id 00U4x000001dPwlEAE; first error: INVALID_FIELD_FOR_INSERT_UPDATE, cannot specify Id in an insert call: [Id]",
            "AsyncApexJobId": "7074x000005tebAAAQ",
            "MethodName": "givenCreateWhenCreateEventThenExpectedListOfEvent",
            "Outcome": "Fail",
            "ApexClass": {
                "attributes": {
                    "type": "ApexClass",
                    "url": "/services/data/v50.0/tooling/sobjects/ApexClass/01p4x000009eXtkAAE"
                },
                "Id": "01p4x000009eXtkAAE",
                "Name": "EventRepositoryTest",
                "NamespacePrefix": null
            },
            "RunTime": 114,
            "FullName": "EventRepositoryTest.givenCreateWhenCreateEventThenExpectedListOfEvent"
        }
    ]
}